docs/resources/mapreduce_cluster.md

```diff
@@ -0,0 +1,354 @@
+    data_volume_count = 1
+  }
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+### Create a hybrid cluster
+```hcl
+data "huaweicloud_availability_zones" "test" {}
+
+variable "cluster_name" {}
+variable "password" {}
+variable "vpc_id" {}
+variable "subnet_id" {}
+
+resource "huaweicloud_mapreduce_cluster" "test" {
+  availability_zone  = data.huaweicloud_availability_zones.test.names[0]
+  name               = var.cluster_name
+  version            = "MRS 1.9.2"
+  type               = "MIXED"
+  component_list     = ["Hadoop", "Spark", "Hive", "Tez", "Storm"]
+  manager_admin_pass = var.password
+  node_admin_pass    = var.password
+  vpc_id             = var.vpc_id
+  subnet_id          = var.subnet_id
+
+  master_nodes {
+    flavor = "c6.2xlarge.4.linux.bigdata"
+    node_number = 2
+    root_volume_type  = "SAS"
+    root_volume_size  = 300
+    data_volume_type  = "SAS"
+    data_volume_size   = 480
+    data_volume_count = 1
+  }
+  analysis_core_nodes {
+    flavor = "c6.2xlarge.4.linux.bigdata"
+    node_number = 2
+    root_volume_type  = "SAS"
+    root_volume_size  = 300
+    data_volume_type  = "SAS"
+    data_volume_size   = 480
+    data_volume_count = 1
+  }
+  streaming_core_nodes {
+    flavor = "c6.2xlarge.4.linux.bigdata"
+    node_number = 2
+    root_volume_type  = "SAS"
+    root_volume_size  = 300
+    data_volume_type  = "SAS"
+    data_volume_size   = 480
+    data_volume_count = 1
+  }
+  analysis_task_nodes {
+    flavor = "c6.2xlarge.4.linux.bigdata"
+    node_number = 1
+    root_volume_type  = "SAS"
+    root_volume_size  = 300
+    data_volume_type  = "SAS"
+    data_volume_size   = 480
+    data_volume_count = 1
+  }
+  streaming_task_nodes {
+    flavor = "c6.2xlarge.4.linux.bigdata"
+    node_number = 1
+    root_volume_type  = "SAS"
+    root_volume_size  = 300
+    data_volume_type  = "SAS"
+    data_volume_size   = 480
+    data_volume_count = 1
+  }
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the MapReduce cluster resource.
+  If omitted, the provider-level region will be used.
+  Changing this will create a new MapReduce cluster resource.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the availability zone in which to create
+  the cluster. Please following [reference](https://developer.huaweicloud.com/en-us/endpoint/?ECS)
+  Changing this will create a new MapReduce cluster resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the MapReduce cluster.
+  The name can contain 2 to 64 characters, which may consist of letters, digits,
+  underscores (_) and hyphens (-).
+  Changing this will create a new MapReduce cluster resource.
+
+* `version` - (Required, String, ForceNew) Specifies the MapReduce cluster version.
+  Changing this will create a new MapReduce cluster resource.
```

Comment:
suggest to add an example

Reply:
add support desctiption for version `MRS 1.9.2`, version `MRS 3.0.5` and version `MRS 3.1.0`.